docs/userguides/deck.md

```diff
@@ -122,3 +128,12 @@ typedef struct deck_driver {
   bool (*test)(void);
 } DeckDriver;
 ```
+
+## Deck Discovery System
+
+The deck discovery system uses a modular backend architecture that supports multiple detection methods. Different discovery backends run during startup:
+
+- **OneWire backend**: The standard method that automatically detects decks with OneWire memory chips
+- **Forced backend**: Allows manually specifying deck names via `CONFIG_DECK_FORCE` for development when physical hardware isn't available
+
+The modular design allows multiple discovery backends to coexist and enables future expansion with additional detection methods.
```

Comment:
Does'nt that mainly repeats what is talked about a bit above?

Reply:
You are completely right! Woopsie!